features/developer_portal/admin/account/personal_details.feature

```diff
@@ -0,0 +1,82 @@
+Feature: Dev Portal Buyer Personal Details
+  As a buyer
+  I want to change my personal details
+
+  Background:
+    Given a buyer logged in to a provider
+
+  # This is needed, otherwise assert_flash doesn't receive the right message.
+  # Check flash-buyer.js
+  @javascript
+  Scenario: Buyer doesn't use current password
+    Given the buyer wants to edit their personal details
+    When the buyer edits their personal details
+    Then they should not be able to edit their personal details
```

Comment:
I think this scenario is missing a step:
_ Given the buyer wants to edit their personal details, when they do, then should not be able to do it._
It doesn't make a lot of sense to me. IMO there should be 2 scenarios: one when the buyer update their details (best case scenario) and another when the password validation fails. Since the current password is a required field, it can be tested in the "form is incomplete" scenario.